hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterHeartbeatManager.java

```diff
@@ -274,14 +282,14 @@ private void onInvalidHeartbeat(MembersViewMetadata senderMembersViewMetadata) {
                 clusterService.sendExplicitSuspicion(senderMembersViewMetadata);
             }
         } else {
-            Address masterAddress = clusterService.getMasterAddress();
-            if (clusterService.getMembershipManager().isMemberSuspected(masterAddress)) {
+            MemberImpl master = clusterService.getMember(clusterService.getMasterAddress());
```

Comment:
Is it legitimate to see null `master` here? if yes, this change can lead NPE.

Reply:
this method is called under the cluster service lock when the member has joined to the cluster. Because if this, `master` cannot be null here I think.